main.cc

```diff
@@ -0,0 +1,210 @@
+#include <iostream>
+#include <string.h>
+#include <cassert>
+
+using namespace std;
```

Comment:
Not in a namepsace/global scope, please!

Reply:
In the next exercises it is allowed... so I allowed myself in advance :) I promise that it just for this exercise...